public/Install-DbaMaintenanceSolution.ps1

```diff
@@ -93,14 +93,14 @@ function Install-DbaMaintenanceSolution {
          https://dbatools.io/Install-DbaMaintenanceSolution
 
     .EXAMPLE
-        PS C:\> Install-DbaMaintenanceSolution -SqlInstance RES14224 -Database DBA -CleanupTime 72
+        PS C:\> Install-DbaMaintenanceSolution -SqlInstance RES14224 -Database DBA -InstallJobs -CleanupTime 72
 
         Installs Ola Hallengren's Solution objects on RES14224 in the DBA database.
         Backups will default to the default Backup Directory.
         If the Maintenance Solution already exists, the script will be halted.
```

Comment:
Do we want to add here that the jobs will be created?

Reply:
We would then have to add it to all of the examples. And "InstallJobs" is quite obvious, I think. So I would vote for no.